lib/ansible/modules/core

```diff
@@ -1 +1 @@
-Subproject commit 85ddb1b90232dbd68798e9b2d7dafa5689a1d30e
```

Comment:
@bcoca: Thanks for your patience but have no idea how to do that ...
Maybe I can refork my repositories and reapply my work against it.
In this case, I'm also wondering if I can split this PR into 2 differents one: 1 for the filter getmountfrompath (are you OK with this name?), and another for the size converter?